modules/ob-defining-strategy-parameters.adoc

```diff
@@ -0,0 +1,45 @@
+// This module is included in the following assembly:
+//
+// builds/configuring-build-strategies.adoc
+
+:_content-type: CONCEPT
+[id="ob-defining-strategy-parameters_{context}"]
+= Defining strategy parameters
+
+You can define strategy parameters in a `BuildStrategy` or `ClusterBuildStrategy`CR and set or modify the values of those parameters in your `Build` or `BuildRun` CR. You can configure or modify strategy parameters at build time when creating your build strategy.
+
+Before defining parameters for your strategy, understand the following points:
+
+* Define a list of parameters in the `spec.parameters` field of your build strategy CR. Each list contains a name, a description, a type, and optionally a default value, or defaults values for an array type. If no default value is set, then you must define a value in the `Build` or `BuildRun` CR.
+* Specify a parameter in the strategy steps by using the syntax `type=string: $(params.your-parameter-name)`. You can set a value for the `your-parameter-name` parameter in your `Build` or `BuildRun` CR that references your strategy. You can use the following string parameters in the `spec.buildSteps` field of your build strategy:
```

Comment:
using the syntax for `type=string: $(params.your-parameter-name)` (missing for)
I found the organisation of this paragraph a bit hard to understand
I would put:

Specify a parameter in the strategy steps by using this syntax for `type=string`:` $(params.your-parameter-name)`. You can use string parameters in all places in the `spec.buildSteps` field of your build strategy.
You can set a value for the `your-parameter-name` parameter in your `Build` or `BuildRun` CR that references your strategy.
Some example scenarios are:

Reply:
@jkhelil @apoorvajagtap just want to be sure whether we need to use `spec.buildSteps` or `spec.steps`?  The later suggestions replace all `buildSteps` instances with just `steps`.

Reply:
With `v1beta1` it should be `spec.steps` ([ref](https://github.com/shipwright-io/build/blob/main/pkg/apis/build/v1beta1/buildstrategy.go#L21))
([v1alpha1 ref](https://github.com/shipwright-io/build/blob/main/pkg/apis/build/v1alpha1/buildstrategy.go#L21))
@jkhelil could you also please confirm this (just to be sure that I am not misinterpreting)?